tests/integration/targets/vm/tasks/01_main.yml

```diff
@@ -62,16 +62,6 @@
             disk_slot: 0
           - type: nic
             nic_vlan: 1
-        cloud_init:
```

Comment:
This was moved to another integration test, since cloud_init wasn't working correctly at the time, this test was ignoring the cloud_init value.
To keep the integrity of the test the same as it was before the change, I removed it from here and made a separate test for it.